source/camera/CameraManager.h

```diff
@@ -62,7 +66,7 @@ class CameraManager
         fullStop
     };
 
-    CameraManager(Ogre::SceneManager* sceneManager, TileContainer* gameMap, Ogre::RenderWindow* renderWindow);
+    CameraManager(Ogre::SceneManager* sceneManager, GameMap* gameMap, Ogre::RenderWindow* renderWindow);
```

Comment:
ok, then it is a random choice. Hopefully, GameMap is compatible with TileContainer. Seems like luck is on our side...